src/lib/components/event/event-details-row-expanded.svelte

```diff
@@ -40,7 +40,7 @@
         {inline}
       />
     </div>
-  {:else if shouldDisplayAsExecutionLink(key)}
+  {:else if shouldDisplayAsExecutionLink(key) && namespace && workflow}
```

Comment:
What do you think about passing `$page.params` to `shouldDisplayAsExecutionLink` and checking for `namespace` and `workflow` there? It seems like they're needed wherever this is currently being called and I see you've done something similar with `shouldDisplayParentWorkflowLink`.